file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json

```diff
@@ -1,9 +1,8 @@
 {
   "formatVersion": "1",
-    "header": {
+  "header": {
```

Comment:
looks like the indentation is off?

Reply:
To me it looks fine. 
Indendation is 2 spaces per level.
FormatVersion and header are the same level, so there should be no indentation

Reply:
oh, okay